homeassistant/components/led_ble/manifest.json

```diff
@@ -3,7 +3,7 @@
   "name": "LED BLE",
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/led_ble/",
-  "requirements": ["bluetooth-data-tools==0.2.0", "led-ble==1.0.0"],
+  "requirements": ["bluetooth-data-tools==0.3.0", "led-ble==1.0.0"],
```

Comment:
This is already merged in dev?

Reply:
> This is already merged in dev?

That was bumped in another PR (just one bellow it): https://github.com/home-assistant/core/pull/82224

Reply:
Yes. So this isn't needed anymore then.

Reply:
That works as well.  closing this.   👍